src/test/java/gumtree/spoon/AstComparatorTest.java

```diff
@@ -1535,7 +1534,7 @@ public void test_issue201812() throws Exception {
 		// https://github.com/GumTreeDiff/gumtree/issues/120
 		CtClass c1 = Launcher.parseClass(" class BehaviorCall implements Call{\n"
 				+ "final AtomicReference failureRef = new AtomicReference<>();\n"
-				+ "final CountDownLatch latch = new CountDownLatch(1);\n" + "\n" + " enqueue(new Callback<T>() {\n"
```

Comment:
why do we need this change?

Reply:
The test asserts that there is [only one root operation](https://github.com/SpoonLabs/gumtree-spoon-ast-diff/pull/179/files/e9e3f6fc019ff28336021a1ac51ac836395940c9#diff-cf51a7b65924ec8a2c187e4b7b552f8fe6c41f239accdbb3ce3da5fcccb08170R1553). However, the changes in this PR compute diff of type argument nodes too. Since `c1` has `T` and `c2` does not, it resulted in another root operation - `Delete TypeParameterReference T`.

Reply:
ok. then it's a better practice to change the assert than the test resource.

Reply:
@monperrus resolved this.